### PR TITLE
x873 - Add external id page

### PR DESCRIPTION
**Pull request by BenTopping**
Closes #

Changes proposed in this pull request:

* New labwork page under Original Sample area - Add External ID
* New icon - Paste icon
* New paste box component

Notes: Testing is pretty low on this one because of cypress's issues with virtualising the window making the paste box very flaky to test

### CODE REVIEW

File: src/pages/AddExternalID.tsx [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+                      id="save"
+                      disabled={formLocked}
+                      className="whitespace-nowrap"
+                      action={"primary"}
+                      type="submit"
+                    >
+                      Save
+                    </BlueButton>
+                  </div>
+                </div>
```
[the remark this answers is not in the document]
Comment (BenTopping) [reply src/pages/AddExternalID.tsx:147]: OK will have a look at this

File: src/components/PasteRestrictedBox.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+          className="px-5 py-3 flex items-center rounded-r-md border-l-0 border text-white bg-sdb-400 focus:outline-none"
+          onClick={() => { handlePaste() }}
+          type={"button"}
+        >
+          <PasteIcon className="block h-5 w-5"/>
+        </button>
+      </div>
+      { value === "" &&
+        <MutedText>Paste an External ID from your clipboard above</MutedText>
+      }
```
[the remark this answers is not in the document]
Comment (seenanair) [reply src/components/PasteRestrictedBox.tsx:64]: Yes, cool. Thank you

File: src/components/PasteRestrictedBox.tsx [hunk 3]
```diff
@@ -18,14 +18,24 @@ export default function PasteRestrictedBox({
...
-      }
-    )
+    try {
+      await navigator.clipboard.readText().then(
+        (text) => {
+          setValue(text)
+          onChange?.(text)
+        }
+      )
+    } catch (error) {
```
Comment (seenanair) [inline src/components/PasteRestrictedBox.tsx:31]: Cool. That looks good, So it will not block users with other browser that doesn't support Paste 👍

**Review by seenanair — APPROVED** [review]
It looks great, similar to other pages 👍  Thank you

**Pull request merged**